nixos/modules/services/web-apps/dolibarr.nix

```diff
@@ -1,11 +1,11 @@
 { config, pkgs, lib, ... }:
 let
-  inherit (lib) any boolToString concatStringsSep isBool isString literalExpression mapAttrsToList mkDefault mkEnableOption mkIf mkOption optionalAttrs types;
+  inherit (lib) any boolToString concatStringsSep isBool isString mapAttrsToList mkDefault mkEnableOption mkIf mkMerge mkOption optionalAttrs types;
```

Comment:
We should just remove this then we don't need to maintain it either

Reply:
IMHO, this can be cleaned up elsewhere if needed at all, but I prefer to avoid `with X;` as it break static analysis in my text editor